rosonwindows_pkgs/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(rosonwindows_pkgs)

```

Comment:
Since we also do work on linux, why not call it msft_ros_pkgs? That way we can host both linux and windows ones